### PR TITLE
Update BinaryTarget

### PR DESCRIPTION
**Pull request by Ignelios**
Updating to latest MaasCore.

### CODE REVIEW

**Comment by justasm** [comment]
Was the checksum for dev06 incorrect?

**Comment by Ignelios** [comment]
> Was the checksum for dev06 incorrect?

Yes it was, i don't know why but it was wrong. cc @mikasdo do you remember how you created checksum ? 🤔

**Comment by mikasdo** [comment]
🤔  i used [swift package compute-checksum path/to/MyFramework.zip](https://developer.apple.com/documentation/swift_packages/distributing_binary_frameworks_as_swift_packages)

**Comment by justasm** [comment]
I'm wondering whether it would be enough to update the checksum for dev06, what changes are in dev07?

File: Package.swift [hunk 1]
```diff
@@ -94,8 +94,8 @@ private extension Environment {
             return [
                 .binaryTarget(
                     name: "MaasCore",
-                    url: "https://github.com/trafi/maas-components/releases/download/0.1.0-dev06/MaasCore.xcframework.zip",
-                    checksum: "d22ef85f0be67f6bf54267d4e1924fedc94ad54377281db250d2465e8e1b8448"),
+                    url: "https://github.com/trafi/maas-components/releases/download/0.1.0-dev07/Core.xcframework.zip",
+                    checksum: "813dd96262bb59a27e0e4d77ce14c8944263b0863dd4947ae3ef65e951dbd984"),
```
Comment (mikasdo) [inline Package.swift:98]: Xcode produces error :( 
`artifact of binary target 'MaasCore' failed download: invalid status code 404`

**Pull request closed without merging**